src/site/content/en/blog/bfcache/index.md

```diff
@@ -303,7 +303,15 @@ onAllChangesSaved(() => {
 {% endCompare %}
 
 
-### Avoid window.opener references
+### Do not serve pages with `Cache-control: no-store`
+
+`Cache-control: no-store` is an HTTP header web servers can set on responses to ensure the response is not stored in any cache. When this is set on the page itself (as opposed to any subresource), this means that page will not be included the bfcache. There is [a proposal to change this](https://bugs.chromium.org/p/chromium/issues/detail?id=1228611) but at present any pages sending this are ineligible to use the bfcache.
```

Comment:
```suggestion
`Cache-Control: no-store` is an HTTP header web servers can set on responses that instructs the browser not to store the response in any cache. When this is set on the page itself (as opposed to any subresource), it means that page will not be eligible for bfcache. There is [a proposal to change this behavior](https://github.com/fergald/explainer-bfcache-ccns/blob/main/README.md) but at present any pages using `Cache-control: no-store` are not eligible for bfcache.
```

Modified the wording a bit for clarity. Also, FYI, Responses with `no-store` can be stored in the [Cache Storage](https://developer.mozilla.org/en-US/docs/Web/API/CacheStorage), so I wouldn't use "ensures" here.